packages/gatsby-recipes/src/renderer/render.js

```diff
@@ -286,14 +292,13 @@ const render = (recipe, cb, inputs = {}, isApply, isStream, name) => {
     }
   }
 
-  const throttledRenderResources = lodash.throttle(renderResources, 30, {
+  const throttledRenderResources = lodash.throttle(renderResources, 100, {
     trailing: false,
   })
 
   queue.on(`task_finish`, function (taskId, r, stats) {
     throttledRenderResources()
 
-    const resources = transformToPlan(result)
     emitter.emit(`update`, resources)
```

Comment:
missing `resources` here ... 

it shows no error here because before the `resources` variable shadowed the `import resources from "../resources"` 

(the same variable shadowing exists also in `renderResources`)

Reply:
ah thanks for this, fixing